lib/hammer_cli_foreman/user.rb

```diff
@@ -11,6 +11,7 @@ class ListCommand < HammerCLIForeman::ListCommand
         field :login, _("Login")
         field :full_name, _("Name")
         field :mail, _("Email")
+        field :mail_enabled, _("Email enabled")
```

Comment:
I guess we shouldn't show this in list command, since the table is already big enough :)

Let's simply stick with the UI:

![ScreenShot-1640017778048](https://user-images.githubusercontent.com/32508194/146800338-ceb88ea4-27d7-4b2c-84a6-38bb3b1394c9.png)